plugins/org.eclipse.che.editor.eclipseide/0.0.1/meta.yaml

```
@@ -0,0 +1,8 @@
+id: org.eclipse.che.editor.eclipseide
+version: 0.0.1
+type: Che Editor
+name: eclipse-ide
+title: Eclipse IDE (in browser using Broadway) as editor for Eclipse Che
+description: Eclipse IDE
+icon: https://github.com/eclipse/eclipse.platform/raw/master/platform/org.eclipse.sdk/eclipse256.png
+url: https://github.com/mickaelistria/che-editor-eclipseide/releases/download/untagged-267df4c0cbc3643d425b/che-editor-plugin.tar.gz
```

Hello, here is a proposed change
```suggestion
url: https://github.com/ws-skeleton/che-editor-eclipseide/releases/download/0.0.1/che-editor-plugin.tar.gz
```

Docker Image is updated to match openshift.io security (root access not allowed) (port number is also changed to be > 1024)
Also update Eclipse .tgz downloaded, squash layers to have smaller image than the original one

Update as well the meta.yaml to match new port number and avoid secured layer (else websocket of broadway is not working with this extra layer)

---

ok!